config/default.yml

```diff
@@ -5186,6 +5186,11 @@ Style/SingleLineBlockParams:
         - acc
         - elem
 
+Style/SingleLineDoEnd:
```

Comment:
I think there's a style guide entry that you can refer here.

Reply:
Sure! I've opened https://github.com/rubocop/ruby-style-guide/pull/922 to add a new rule for this one.